components/pkg-export-kubernetes/src/manifest.rs

```diff
@@ -28,20 +28,39 @@ use topology::Topology;
 use manifestjson::ManifestJson;
 use bind;
 
+/// Represents a Kubernetes manifest.
 #[derive(Debug, Clone)]
 pub struct Manifest {
+    /// The service name in the Kubernetes cluster.
     pub metadata_name: String,
+    /// The service name in the Habitat ring.
     pub habitat_name: String,
+    /// The docker image.
     pub image: String,
+    /// The amount of Services that should start in the Habitat ring.
```

Comment:
Maybe instead of "in the Habitat ring", write "in the service group"?

Reply:
isn't the "ring" a similar concept to "cluster" of k8s?

Reply:
It's better not to make parallels, as the worlds are really quite different :)